cmd/cstorvolumeclaim/controller.go

```diff
@@ -260,8 +247,12 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 	if err != nil {
 		return nil, err
 	}
+
+	// update the cstorvolume reference, phase as "Bound" and desired
+	// capacity
 	cvc.Spec.CStorVolumeRef = volumeRef
-	cvc.Status.Phase = "Bound"
+	cvc.Status.Phase = apis.CStorVolumeClaimPhaseBound
+	cvc.Status.Capacity = cvc.Spec.Capacity
```

Comment:
let us not set the Status.Capacity to Spec.Capacity.. let the resize workflow set the Status.Capacity

Reply:
its good to update the `status.capacity` at the bound time( initial create process) to complete the provisioning, we should differentiate this from the resize flow.